config.go

```diff
@@ -138,13 +138,23 @@ func loadConfigBytes(conf interface{}, data []byte, custom customFunc, unmarshal
 
 // Delims sets the action delimiters to the specified strings.
 func Delims(left, right string) {
-	envRepTpl.Delims(left, right)
+	DefaultLoader.Delims(left, right)
+}
+
+// Funcs adds the elements of the argument map.
+// Caution: global settings are overwritten. can't go back.
+func Funcs(funcMap template.FuncMap) {
+	DefaultLoader.Funcs(funcMap)
 }
 
-var envRepTpl *template.Template
+var DefaultLoader *Loader
```

Comment:
DefaultLoader must be a private variable. Because it should not be overwritten from out of a package.